cohttp-server-lwt-unix/src/cohttp_server_lwt_unix.mli

```diff
@@ -0,0 +1,28 @@
+module Body : sig
+  module Encoding : sig
+    type t
+
+    val fixed : int64 -> t
+    val chunked : t
+  end
+
+  type t
+
+  val string : ?encoding:Encoding.t -> string -> t
+end
+
+module Context : sig
+  type t
+
+  val request : t -> Http.Request.t
+  val read_body : t -> string Lwt.t
+  val discard_body : t -> unit Lwt.t
+  val respond : t -> Http.Response.t -> Body.t -> unit Lwt.t
+end
+
+type t
+
+val create : ?on_exn:(exn -> unit) -> (Context.t -> unit Lwt.t) -> t
```

Comment:
This kind of callback allows us to send a response and then continue doing something. That's not possible with a `request -> response` style of callback.